src/main/java/seedu/address/logic/parser/CliSyntax.java

```diff
@@ -5,15 +5,9 @@
  */
 public class CliSyntax {
 
-    /* Prefix definitions */
-    public static final Prefix PREFIX_NAME = new Prefix("n/");
-    public static final Prefix PREFIX_PHONE = new Prefix("p/");
-    public static final Prefix PREFIX_EMAIL = new Prefix("e/");
-    public static final Prefix PREFIX_ADDRESS = new Prefix("a/");
-    public static final Prefix PREFIX_TAG = new Prefix("t/");
-
     //ModBook Prefixes
```

Comment:
prob can remove this? since its only modbook now

Reply:
Should be removed (and thus in red)